src/en/about/guide.md

```diff
@@ -1,57 +1,105 @@
 
-Type | Or | … to Get 
---- | --- | ---
-\*Italic\* | \_Italic\_  | _Italic_ 
-\*\*Bold\*\* | \_\_Bold\_\_ | **Bold** 
-\# Heading 1 | Heading 1 <br> ========= | <h4>Heading 1</h4>
-\## Heading 2 | Heading 2 <br>--------- | <h5>Heading 2</h5>
-\[Link\](http://a.com) | \[Link\]\[1\]<br>⋮ <br>\[1\]: http://b.org | [Link](https://commonmark.org/) 
-!\[Image\](http://url/a.png) | !\[Image\]\[1\]<br>⋮ <br>\[1\]: http://url/b.jpg | ![Markdown](https://commonmark.org/help/images/favicon.png) 
-\> Blockquote | | <blockquote>Blockquote</blockquote>
-\* List <br>\* List <br>\* List | \- List <br>\- List <br>\- List <br> | *   List <br>*   List <br>*   List <br>
-1\. One <br>2\. Two <br>3\. Three | 1) One<br>2) Two<br>3) Three | 1.  One<br>2.  Two<br>3.  Three
-Horizontal Rule <br>\--- | Horizontal Rule<br>\*\*\* | Horizontal Rule  <br><hr>
-\`Inline code\` with backticks | |`Inline code` with backticks 
-\`\`\`<br>\# code block <br>print '3 backticks or'<br>print 'indent 4 spaces' <br>\`\`\` | ····\# code block<br>····print '3 backticks or'<br>····print 'indent 4 spaces' | \# code block <br>print '3 backticks or'<br>print 'indent 4 spaces'
-::: spoiler hidden or nsfw stuff<br>*a bunch of spoilers here*<br>::: | | <details><summary> hidden or nsfw stuff </summary><p><em>a bunch of spoilers here</em></p></details>
-Some ~subscript~ text | | Some <sub>subscript</sub> text
-Some ^superscript^ text | | Some <sup>superscript</sup> text
+| Type | Or  | … to Get |
+| --- | --- | --- |
+| \*Italic\* | \_Italic\_ | *Italic* |
+| \*\*Bold\*\* | \_\_Bold\_\_ | **Bold** |
+| \# Heading 1 | Heading 1<br>========= | #### Heading 1 |
+| \## Heading 2 | Heading 2<br>\-\-\-\-\-\-\-\-\- | ##### Heading 2 |
+| \[Link\](http://a.com) | \[Link\]\[1\]<br>⋮<br>\[1\]: http://b.org | [Link](https://commonmark.org/) |
+| !\[Image\](http://url/a.png) | !\[Image\]\[1\]<br>⋮<br>\[1\]: http://url/b.jpg | ![Markdown](https://commonmark.org/help/images/favicon.png) |
+| \> Blockquote |     | > Blockquote |
+| \* List<br>\* List<br>\* List | \- List<br>\- List<br>\- List | \* List<br>\* List<br>\* List |
+| 1\. One<br>2\. Two<br>3\. Three | 1) One<br>2) Two<br>3) Three | 1\. One<br>2\. Two<br>3\. Three |
+| Horizontal Rule<br>\-\-\- | Horizontal Rule<br>\*\*\* | Horizontal Rule<br><br>* * * |
+| \`Inline code\` with backticks |     | `Inline code` with backticks |
+| ```<br>\# code block<br>print '3 backticks or'<br>print 'indent 4 spaces'<br>``` | ····# code block<br>····print '3 backticks or'<br>····print 'indent 4 spaces' | \# code block<br>print '3 backticks or'<br>print 'indent 4 spaces' |
+| ::: spoiler hidden or nsfw stuff<br>*a bunch of spoilers here*<br>::: |     | <details><summary>hidden or nsfw stuff</summary><br><br>*a bunch of spoilers here*<br><br></details> |
+| Some ~subscript~ text |     | Some <sub>subscript</sub> text |
+| Some ^superscript^ text |     | Some <sup>superscript</sup> text |
 
 [CommonMark Tutorial](https://commonmark.org/help/tutorial/)
-
```

Comment:
This file is quite long now. Maybe it would be good to make a new category "users", with files along the lines of "getting started", "navigating lemmy", "formatting". Or something like that. Maybe ranking.md should also go in that category. Then it might be logical to have one category per role (users, moderators, administrators).

Reply:
I don't mind a long file, as long as the headings are organized. And we could always use `markdown-toc` to generate a TOC if necessary.